samples/bluetooth/mesh/light_lc/README.rst

```diff
@@ -0,0 +1,189 @@
+   |               | Gen. DTT Server               | Light Control Setup Server |
+   +---------------+-------------------------------+----------------------------+
+   |               | Gen. Power OnOff Server       |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Gen. Power OnOff Setup Server |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Light Lightness Server        |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Light Lightness Setup Server  |                            |
+   +---------------+-------------------------------+----------------------------+
+
+The models are used for the following purposes:
+
+The first element consists of a Config Server and a Health Server. The Config Server allows configurator devices to configure the node
+remotely, and the Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to
+the device. These callbacks trigger blinking of the LEDs.
+
+The eight models contained within the second element is the result of a single implementation of the Light Lightness Server. These
+models are bound internally to the light source hardware you wish to control (in this case the first LED of the DK).
+External control of the light source can be achieved directly by using either the Gen. Level Server or the Gen. OnOff Server
+in the second element.
+
+The three models contained within the third element is the result of a single implementation of the Light Control Server.
+Together they establish a controller that can, if enabled, take control over a Light Lightness Server instance, where the
+controller decides parameters such as fade time, lighting levels for different states, and inactivity timing. In this sample
+the Light Control Server is bound internally to the Light Lightness Server instance in the second element and enabled by default on boot up.
+External control of the light source associated with the Light Lightness Server can be achieved indirectly by the Light Control
+Server by using the Gen. OnOff Server in the third element.
+
+For a more detailed description checkout the documentation for the :ref:`bt_mesh_lightness_srv_readme` and
+:ref:`bt_mesh_light_ctrl_srv_readme`.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` and the
+:ref:`zephyr:pwm_api` API to control the LEDs on the board.
+
+Requirements
+************
+
+* One of the following development kits:
+
+  * |nRF52DK|
+  * |nRF52840DK|
+
+* Smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
+
+  * `nRF Mesh mobile app for Android`_
+  * `nRF Mesh mobile app for iOS`_
+
+User interface
+**************
+
+Buttons:
```

Comment:
I know that this is reused, so it does not have to be changed (at least not here only). Just wondered why the columns (:) are used to begin with here. Don't think they're needed.

Reply:
Your right, it is a knock-off  😅